cmd/livepeer/livepeer.go

```diff
@@ -89,7 +89,9 @@ func main() {
 	}
 
 	if *rinkeby {
+		*datadir = *datadir + "/rinkeby"
 		if !*offchain {
+			*datadir = *datadir + "/offchain"
```

Comment:
This would result in `datadir/rinkeby/offchain` when it should be `datadir/offchain` -- same for mainnet

Reply:
No worries -- this area of the code really needs improvement, it's difficult to reason about. Your changes look good now!